### ISSUE TITLE
Nether Ambience not working

### ISSUE THREAD

**Issue by Zombieh** [OreCruncher/DynamicSurroundings#592]
**Mod Version:**
DynamicSurroundings-1.12.2-3.6.1.0

**Forge Version:**
14.23.5.2854

**Link to client log:**
https://pastebin.com/QKb5JtSY

**Description:**
Upon entering the Nether, there is supposed to be ambient audio that plays in the background, but it doesn't. The ambient audio DOES work in the end, but it does not work in the nether. This was done with only OreLib and DynamicSurroundings installed on the forge client.

**Comment by OreCruncher** [comment]
I haven't been able to duplicate this issue.  Does the sound not even play if you are moving around the nether?

**Comment by Zombieh** [comment]
Nope, doesn't play no matter where I move in the nether strangely enough... I've even tried to do fresh installs and new worlds but to no avail.

**Comment by OreCruncher** [comment]
Can you do several things?
1. Link your dsurround.cfg file.
2. Before starting your client, in the config set "Enable Debug Logging" to true, and "Debug Flag Mask" to 1.  The do what you normally do to repro the issue.  The link that client log.

**Comment by Zombieh** [comment]
Sure thing, I have no idea why, but when I pasted the client log into pastebin it deemed it as offensive, but yeah. Here's the client log and config I have:

**Client Log**
https://pastebin.com/yR5muUCk

**Config**
https://pastebin.com/Dh8ShSuH

**Comment by OreCruncher** [comment]
Thanks!  No weird errors listed in the log, and it looks like things configured correctly.  I am not seeing any attempt to play the nether dimension sound, so I need to dig into those triggers to figure out what is going on.

**Comment by Zombieh** [comment]
Yep, it plays both the heartbeat and the stomach rumble sounds.

**Comment by Zombieh** [comment]
This work?:
https://imgur.com/a/mNJMkuL

**Comment by OreCruncher** [comment]
Yes it does.  It does show that the nether dimension sound is playing at a volume of 0.3 which is what the volume is configured for.  Not sure why are you are not hearing it.  What is the Biome volume slider set to in Minecraft's sound configuration page (where you set the volume scale for each of the sound categories)?

**Comment by Zombieh** [comment]
It is set to 100% volume.

**Comment by disastercube** [comment]
I've got the same problem with the latest version.

**Comment by OreCruncher** [comment]
@Zombieh, @disastercube that is really weird.  Not sure what to make of it.  Are either of you using a published modpack?

**Comment by disastercube** [comment]
I'm playing with a self created modpack.
But there is something weird happening. Normally the latest version is dependent on OreLib and crashes without MixinBootstrap but in my case when using it with my modpack it crashes if using MixinBootstrap and works fine without (but I need to include it if I'm only using the mod ).
To be clear the sounds are working such as walking but the music is the one that is not playing, and yes the music is working fine in the overworld.

**Comment by disastercube** [comment]
any ideas ?

**Comment by OreCruncher** [comment]
MixinBootstrap supplies Mixin.  If there is another mod supplying Mixin then you don't need it.  To be clear when you refer to music you are talking about the biome sounds, right?  Reason is that where Minecraft is concerned music is something specific and is handled by the Minecraft music player.

**Comment by disastercube** [comment]
Yeah kinda, I'm talking about the sinister music that plays in the nether.

**Comment by Zombieh** [comment]
I personally am not using a Modpack, I installed the mod fresh, with the OreLib and it still wasn't working for me...
